src/tlo/methods/wasting.py

```diff
@@ -1,41 +1,1290 @@
+                                            p=[p['proportion_mam_with_MUAC_[115-125)mm_and_normal_whz'],
+                                               p['proportion_mam_with_MUAC_[115-125)mm_and_-3<=WHZ<-2'],
+                                               p['proportion_mam_with_-3<=WHZ<-2_and_normal_MUAC']])
+
+            if mam_classification == 'mam_by_muac_only':
+                df.at[person_id, 'un_WHZ_category'] = 'WHZ>=-2'
+                df.at[person_id, 'un_am_MUAC_category'] = '[115-125)mm'
+
+            if mam_classification == 'mam_by_muac_and_whz':
+                df.at[person_id, 'un_WHZ_category'] = '-3<=WHZ<-2'
+                df.at[person_id, 'un_am_MUAC_category'] = '[115-125)mm'
+
+            if mam_classification == 'mam_by_whz_only':
+                df.at[person_id, 'un_WHZ_category'] = '-3<=WHZ<-2'
+                df.at[person_id, 'un_am_MUAC_category'] = '>=125mm'
+
+        # Update all other properties equally
+        df.at[person_id, 'un_clinical_acute_malnutrition'] = 'MAM'
+        df.at[person_id, 'un_am_bilateral_oedema'] = False
+        df.at[person_id, 'un_sam_with_complications'] = False
+        df.at[person_id, 'un_am_tx_start_date'] = pd.NaT
+        df.at[person_id, 'un_am_recovery_date'] = pd.NaT
+        df.at[person_id, 'un_am_discharge_date'] = pd.NaT
+        # will start the process again
+        df.at[person_id, 'un_am_treatment_type'] = 'not_applicable'
+
+        # this will clear all wasting symptoms (applicable for SAM, not MAM)
+        self.sim.modules["SymptomManager"].clear_symptoms(
+            person_id=person_id, disease_module=self.module
+        )
+
+
+class HSI_Wasting_SupplementaryFeedingProgramme_MAM(HSI_Event, IndividualScopeEventMixin):
+    """
+    This is the supplementary feeding programme for MAM without complications
+    """
+
+    def __init__(self, module, person_id):
+        super().__init__(module, person_id=person_id)
+        assert isinstance(module, Wasting)
+
+        # Get a blank footprint and then edit to define call on resources
+        # of this treatment event
+        the_appt_footprint = self.sim.modules["HealthSystem"].get_blank_appt_footprint()
+        the_appt_footprint['Under5OPD'] = 1  # This requires one outpatient
+
+        # Define the necessary information for an HSI
+        self.TREATMENT_ID = 'Undernutrition_Feeding_Supplementary'
+        self.EXPECTED_APPT_FOOTPRINT = the_appt_footprint
+        self.ACCEPTED_FACILITY_LEVEL = '1a'
+        self.ALERT_OTHER_DISEASES = []
+
+    def apply(self, person_id, squeeze_factor):
+        df = self.sim.population.props
+
+        if not df.at[person_id, 'is_alive']:
+            return
+
+        # Do here whatever happens to an individual during this health system interaction event
+        # ~~~~~~~~~~~~~~~~~~~~~~
+        # Make request for some consumables
+        consumables = self.sim.modules['HealthSystem'].parameters['item_and_package_code_lookups']
```

Comment:
This is done 3 times in different HSIs. It would be nice to put the looking up of consumables in one function and then access them from there as done in the other modules like RTI and COPD. Perhaps this can be done in a separate PR.